sdk/src/client/node_api/core/routes.rs

```diff
@@ -304,6 +304,19 @@ impl ClientInner {
         Ok(OutputMetadata::try_from(metadata)?)
     }
 
+    // Finds output and its metadata by output ID.
+    /// GET /api/core/v3/outputs/{outputId}
+    /// + GET /api/core/v3/outputs/{outputId}/metadata
+    pub async fn get_output_with_metadata(&self, output_id: &OutputId) -> Result<OutputWithMetadata> {
+        let output = Output::unpack_verified(
```

Comment:
this isnt a core route though, its 2 routes combined. shouldnt that be in another file? like core/mod.rs

Reply:
Ok I'll move it